src/Dropzone/README.md

```diff
@@ -61,7 +61,7 @@ the `@symfony/ux-dropzone/src/style.css` autoimport to `false`:
         "@symfony/ux-dropzone": {
             "dropzone": {
                 "enabled": true,
-                "webpackMode": "eager",
+                "fetch": "eager",
```

Comment:
For clarification to others, the `package.json` files themselves in this repo still have `"webpackMode": "eager"`. However, that is just for backwards-compat with stimulus-bridge 1. `symfony/flex` 1.12.0 (thanks to https://github.com/symfony/flex/pull/735) will read the `webpackMode` but actually put `fetch` into your `controllers.json` file.

tl;dr - this fix is correct

Reply:
But the Readme stay confusing because don't tell anything about this.

Maybe a note could be added at least to clarify this.